tests/system_tests_one_router.py

```diff
@@ -3216,5 +3216,86 @@ def run(self):
                 sleep(0.1)
 
 
+class DataConnectionCountTest(TestCase):
```

Comment:
Can we add a test that runs a `qdmanage` command (using the `QdManager` class) and run a query on the connections and make sure that there are actually that many number of connections as calculated by the dataConnectionCount ?